dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -71,7 +71,6 @@
 
         #schooldropdown-block.itemblock{style: "display: none;"}
           = render partial: 'shared/school_info_registration', locals: {form_name: "user[school_info_attributes]", existing_school_info: nil, suppress_scrolling: true}
-          .error_in_field
```

Comment:
Yes, in the workshop registration version of this, there's a single error field, for some reason that wasn't working right for me here, so I changed it a bit. In the React rewrite things will be a little simpler, I hope.